### ISSUE TITLE
Error: Sprockets::FileNotFound: couldn't find flatuipro.js in Travis-CI

### ISSUE THREAD

**Issue by akshatpradhan** [reflection/designmodo-flatuipro-rails#18]
@reflection This gem is not test friendly. Its especially not Travis-CI friendly either. Travis-CI is a hosted continuous integration service that don't allow us root access. We push our code up there through git and they run our tests. 

Here's the error
https://travis-ci.org/akshatpradhan/spoutlets/builds/12443369#L302 where its looking for `flatuipro.js` and its not finding it. 

```
Error: Sprockets::FileNotFound: couldn't find file 'jquery.ui.touch-punch.min'
       (in /home/travis/build/akshatpradhan/spoutlets/app/assets/javascripts/flatuipro.js:1)
Error: Sprockets::FileNotFound: couldn't find file 'jquery.ui.touch-punch.min'
 (in /home/travis/build/akshatpradhan/spoutlets/app/assets/javascripts/flatuipro.js:1)
   at http://127.0.0.1:51875/assets/application.js:1
```

I imagine other continuous integration services would pick up the same problem.

@adamico was able to confirm this as well.

**Comment by kovacs** [comment]
Any word on this problem? Clues on where one might look to try and fix it?

**Comment by akshatpradhan** [comment]
Yeah the solution was to move to this: https://github.com/wingrunr21/flat-ui-sass

The real problem is flat ui's licensing. Designmodo's "open source" version is really just crippleware.
